### ISSUE TITLE
Reproduce the training result

### ISSUE THREAD

**Issue by vqnhat** [kaiwang960112/Challenge-condition-FER-dataset#30]
Hi, thank you for your excellent work!

I am trying to reproduce the training results on the full FER_Plus dataset by using the code (train_attention_rank_loss.py) and pretrained resnet18 model (ijba_res18_native.pth.tar) you provided. I also use the fixed cropping strategy as in your paper (full image + 5 regions). Besides the learning rate, all of the training settings are unchanged.

However, I could not achieve the accuracy mentioned in the paper. Most of the time, the validation and testing accuracy are only around 85% and 83%, respectively. The model can fit pretty well on the training set, though.

Do you have any special preparation for setting up the training or any configuration of the hyperparameters? Your suggestion would be highly appreciated!

**Comment by LiuYeeJ** [comment]
Hello, I have been doing experimental reproducibility recently. I would like to ask you whether you are experimenting directly through the model shared by the author? Because my display file is damaged, and because of my weak foundation, I have not completed the recurrence of the experiment. I want to ask you some questions, you can ask for a fee, I hope you can help me, thank you very much




> Hi, thank you for your excellent work!
> 
> I am trying to reproduce the training results on the full FER_Plus dataset by using the code (train_attention_rank_loss.py) and pretrained resnet18 model (ijba_res18_native.pth.tar) you provided. I also use the fixed cropping strategy as in your paper (full image + 5 regions). Besides the learning rate, all of the training settings are unchanged.
> 
> However, I could not achieve the accuracy mentioned in the paper. Most of the time, the validation and testing accuracy are only around 85% and 83%, respectively. The model can fit pretty well on the training set, though.
> 
> Do you have any special preparation for setting up the training or any configuration of the hyperparameters? Your suggestion would be highly appreciated!

Hello, I have been doing experimental reproducibility recently. I would like to ask you whether you are experimenting directly through the model shared by the author? Because my display file is damaged, and because of my weak foundation, I have not completed the recurrence of the experiment. I want to ask you some questions, you can ask for a fee, I hope you can help me, thank you very much

**Comment by kaiwang960112** [comment]
did you align the images? and you can load  the pre-trained model directly
(using tar)

LiuYeeJ ***@***.***> 於 2021年4月26日週一 下午8:32寫道：

**Comment by vqnhat** [comment]
Thank you for your reply. The face images have been aligned by retina face before training. The pretrained model was loaded directly by torch.load()
